src/scripts.js

```diff
@@ -55,20 +55,27 @@ homeButton.addEventListener('click', displayHomePage);
 favoritePageButton.addEventListener('click', displayFavorites);
 addToCookbookButton.addEventListener('click', toggleCookbookButton);
 cookbookPageButton.addEventListener('click', displayCookbook);
+singleViewFavoriteButton.addEventListener('click', favoriteFromSingleRecipeView);
 
 // FUNCTIONS
 function displayFavorites() {
   hide([favoritePageButton]);
   show([homeButton, filterSection, recipeSection, mainView]);
   displayRecipes(currentUser.favorites);
   recipeCards = document.querySelectorAll('.recipe-card-js');
+  console.log(recipeCards);
   recipeCards.forEach(card => {
-    const button = card.childNodes[3].childNodes[1]
-    button.value = 'favorited';
-    button.classList.add('favorited-state');
+    card.addEventListener('click', removeFromPage);
   });
 }
 
+function removeFromPage() {
+  // if (currentUser.favorites.includes(e.target.closest('section').id.slice(2))) {
```

Comment:
will delete- started going down this path but realized there was a much simpler way to accomplish this